src/blas/wrappers.jl

```diff
@@ -937,15 +937,16 @@ for (fname, elty) in
         function gemm_strided_batched!(transA::Char,
                                transB::Char,
                                alpha::($elty),
-                               A::CuArray{$elty, 3},
```

Comment:
I think when I tried to wrap this I intended to use this as a low level API, now since both CUDAnative and CuArrays changed a lot, maybe we need a more bare wrapper (like a pointer type `CuPtr`) directly wraps the CUBLAS API? then it'd be more elegant to have a higher level wrapper for different Julia array types.

Reply:
Can `NNlib.batched_mul` be this higher-level wrapper? FluxML/NNlib.jl#191 makes it more flexible, and able to dispatch according to the underlying data.

And what can't you do with this wrapper (which works on any AbstractArray for which this pointer exists) which you could do with a different one?